src/utilities/focus.ts

```diff
@@ -154,8 +154,9 @@ export function isElementTabbable(element: HTMLElement): boolean {
     !!element &&
     (element.tagName === 'A' ||
       (element.tagName === 'BUTTON' && !(element as HTMLButtonElement).disabled) ||
-      (element.tagName === 'INPUT' && !(element as HTMLButtonElement).disabled) ||
-      (element.getAttribute && element.getAttribute(IS_FOCUSABLE_ATTRIBUTE) === 'true')));
+      (element.tagName === 'INPUT' && !(element as HTMLInputElement).disabled) ||
+      (element.getAttribute && element.getAttribute(IS_FOCUSABLE_ATTRIBUTE) === 'true')) ||
+      (element.getAttribute && element.getAttribute('tabindex') === '0'));
```

Comment:
I don't think this will work. FocusZone will unset the tabindex after tabbing out, and from that point on will treat the tabbable as a non-tabbable.